test/core/derivatives/problem.py

```diff
@@ -169,9 +172,27 @@ def forward_pass(self, input_requires_grad=False, sample_idx=None):
 
         return input, output, dict(self.module.named_parameters())
 
+    def derivative_prefix_for(self):
+        self.derivative = self.make_derivative()
+        if is_activation(self.derivative):
+            prefix = "act"
+        elif is_layer(self.derivative):
+            prefix = "layer"
+        elif is_pooling(self.derivative):
+            prefix = "pool"
+        elif is_loss(self.derivative):
+            prefix = "loss"
+        else:
+            prefix = ""
+        return prefix
+
```

Comment:
Thought a bit more about it and saw the modified names in the failing travis run. Now I think this does not add sufficient new information. If s.o. is developing a new `MyLayer`, it's possible to filter the tests with
```
pytest -vx . -k MyLayer
``` 
Could you undo this prefixing? I might have missed an argument.